docs/UserGuide.md

```diff
@@ -320,13 +327,32 @@ Furthermore, certain edits can cause the UniLink to behave in unexpected ways (e
 **Filter** | `filter [CONTACT_TYPE]`<br> e.g., `filter WORK`, `filter PERSONAL`
 **Import** | `import`
 **List**   | `list`
-**Switch** | `switch [THEME]`<br> e.g., `switch LIGHT`, `switch DARK`
+**Switch** | `switch THEME`<br> e.g., `switch LIGHT`, `switch DARK`
 **Help**   | `help`
+**Exit**   | `exit`
+
+--------------------------------------------------------------------------------------------------------------------
+
+## Field Constraints
+
+To ensure your commands work as expected, refer to the table below for guidance on each field’s requirements, including format and length. This way, you’ll know exactly how to input information like names, phone numbers, and other fields correctly. We’ve also included examples to make things easier!
+
+| **Field**       | **Constraints**                                                                                                                                                                                                                                                                                                                                                                                                                                                                      | **Examples**                                           |
+|-----------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------|
+| NAME            | - Should only contain alphabets and spaces                                                                                                                                                                                                                                                                                                                                                                                                        | Alice Tan <br> Hubert Koh                              |
+| CONTACT TYPE    | - Should be 'WORK' or 'PERSONAL' <br> - Not case sensitive                                                                                                                                                                                                                                                                                                                                                                                                                           | Work <br> personal                                     |
```

Comment:
can add the 'SCHOOL' contact type as well

Reply:
updated!